react/react-basics/step5.md

```diff
@@ -35,12 +35,12 @@ Use the same variant and properties as step 2.
 &lt;Button variant=&quot;link&quot; isInline>Link to home&lt;/Button&gt;
 </pre>
 
-4) <strong>Add a gutter property to the `Split` component to add more spacing between the buttons.</strong>
+4) <strong>Add a hasGutter property to the `Split` component to add more spacing between the buttons.</strong>

```

Comment:
Not sure which way we want to go with this, but in the button examples, the `isInline` prop has code back ticks, but hasGutter does not. Also we refer to both "prop" (once) and "property" (three times) on step 5. We should probably be consistent with those.

```suggestion
4) <strong>Add a `hasGutter` prop to the `Split` component to add more spacing between the buttons.</strong>

```